### ISSUE TITLE
Sharing cookies between instances

### ISSUE THREAD

**Issue by keis** [uzbl/uzbl#230]
The cookie commands are only stubs for webkit2 and does not do anything.

Can we make them work again?
Some other way of making this work?

Ref #225

**Comment by SirCmpwn** [comment]
:+1: I have to leave uzbl until this is fixed.

**Comment by keis** [comment]
If you want something usable today look at the master branch or the 0.9.0 release

**Comment by SirCmpwn** [comment]
Yeah, I figured as much. Will probably hop in and see if I can help on next.

**Comment by keis** [comment]
Talked a bit to the #webkitgtk+ people a while back and it was pointed out that in the modern web cookies is the least of our problems between local storage, websql, appcache and what have you.

hacking up a way of sharing cookies again is not likely to cut and we really need a way of sharing the entire network process. On a high-level this should be perfectly doable in a multiprocess way but the webkit repo is a dark and scary place.

Another options is to multiplex multiple uzbls inside one process, I'm not a big fan but it's perhaps the easier way forward and even that is a lot of work.

**Comment by mathstuf** [comment]
We can do that once we remove the global `uzbl` object and make it passed around everywhere in the `user_data` field of signal callbacks.
